streams/de.m3u

```diff
@@ -470,21 +468,47 @@ https://edge14.vedge.infomaniak.com/livecast/ik:wir24tv_hms/manifest.m3u8
 https://a.jsrdn.com/broadcast/ab14783a09/+0000/c.m3u8
 #EXTINF:-1 tvg-id="WWTV.de",WW TV (720p)
 https://live2.telvi.de/hls/wwtv_hd720.m3u8
-#EXTINF:-1 tvg-id="ZDF.de",ZDF [Geo-blocked]
+#EXTINF:-1 tvg-id="ZDF.de",ZDF (720p) [Geo-blocked]
 https://zdf-hls-15.akamaized.net/hls/live/2016498/de/high/master.m3u8
-#EXTINF:-1 tvg-id="ZDFinfo.de",ZDFinfo [Geo-blocked]
+#EXTINF:-1 tvg-id="ZDFinfo.de",ZDFinfo (720p) [Geo-blocked]
 https://zdf-hls-17.akamaized.net/hls/live/2016500/de/high/master.m3u8
-#EXTINF:-1 tvg-id="ZDFneo.de",ZDFneo [Geo-blocked]
+#EXTINF:-1 tvg-id="ZDFneo.de",ZDFneo (720p) [Geo-blocked]
 https://zdf-hls-16.akamaized.net/hls/live/2016499/de/high/master.m3u8
 #EXTINF:-1 tvg-id="ZweiMusicTelevision.de",ZWEI2 Music (1080p)
-https://serve-first.folxplay.tv/folx/ch-2/master.m3u8
+https://cdne.folxplay.tv/folx-trz/streams/ch-2/master.m3u8
 #EXTINF:-1 tvg-id="",Isar TV (1080p) [Not 24/7]
 https://isar-tv.iptv-playoutcenter.de/isar-tv/isar-tv.stream_1/playlist.m3u8
 #EXTINF:-1 tvg-id="",Magenta Musik 360 2 (1080p)
 https://streaming.magenta-musik-360.de/csm/573871/magentamusik2/index.m3u8
-#EXTINF:-1 tvg-id="",OK Fulda
+#EXTINF:-1 tvg-id="",OK Fulda (720p)
 https://s.ok54.de/mok-fu/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="",ONE1 Slovenija (1080p)
-https://serve-first.folxplay.tv/folx/ch-6/master.m3u8
-#EXTINF:-1 tvg-id="",PunktUm Hettstedt (1080p)
-https://58bd5b7a98e04.streamlock.net/medienasa-live/_definst_/mp4:punktum_high/playlist.m3u8
+#EXTINF:-1 tvg-id="OneAdria.de",One Adria (1080p)
+https://cdne.folxplay.tv/folx-trz/streams/ch-6/master.m3u8
+#EXTINF:-1 tvg-id="OKFlensburg.de",OK Flensburg (576p) 
+https://live-cdn.oksh.de/play/hls/flensburgtv/index.m3u8
+#EXTINF:-1 tvg-id="DerAktionaerTV.de",Der Aktionär TV (720p) 
+https://live2.telvi.de/hls/aktionaertv.m3u8
+#EXTINF:-1 tvg-id="DeluxeRap.de",Deluxe Rap (1080p) [Geo-blocked]
+https://sdn-global-live-streaming-packager-cache.3qsdn.com/65183/65183_264_live.m3u8
+#EXTINF:-1 tvg-id="DeluxeFlashback.de",Deluxe Flashback (1080p) [Geo-blocked]
+https://sdn-global-live-streaming-packager-cache.3qsdn.com/65185/65185_264_live.m3u8
+#EXTINF:-1 tvg-id="DeluxeDance.de",Deluxe Dance (1080p) [Geo-blocked]
+https://sdn-global-live-streaming-packager-cache.3qsdn.com/64733/64733_264_live.m3u8
+#EXTINF:-1 tvg-id="AstroTV.de",AstroTV (1080p) [Not 24/7]
+https://ythls.onrender.com/channel/UCwS3E2NaN3uorr9wf2_i6Cg.m3u8
+#EXTINF:-1 tvg-id="PearlTV.de",Pearl TV (2160p) [Not 24/7]
+https://ythls.onrender.com/channel/UCp0urejlDaYz4G8xpTxdIvw.m3u8
+#EXTINF:-1 tvg-id="AristoTV.de",Aristo TV (360p)
+https://ma.anixa.tv/clips/stream/aristo/index.m3u8
+#EXTINF:-1 tvg-id="AnixePlus.de",Anixe + (360p)
+https://ma.anixa.tv/clips/stream/anixesd/index.m3u8
```

Comment:
https://ma.anixa.tv/clips/stream/anixehd/index.m3u8 is Eazzzy TV, if you want to add it.

Reply:
Is already added.